pkg/apis/serving/v1alpha1/framework.go

```diff
@@ -41,6 +42,10 @@ var (
 	DefaultCPURequests    = resource.MustParse("1")
 )
 
+func (m *ModelSpec) MapSourceUri() (sourceURI string, localPath string, ok bool) {
```

Comment:
Would love to see this collapsed as GetModelUri and avoid the extra config

Reply:
collapsed to GetModelUri